modules/apps/callbacks/ibc_middleware_test.go

```diff
@@ -119,6 +119,18 @@ func (s *CallbacksTestSuite) TestSendPacket() {
 			false,
 			nil,
 		},
+		{
+			"success: multiple denoms",
```

Comment:
nit: I would move this test case up, just right after `success`.

Reply:
Oh yeah, I didn't realize the previous case a form of failure. I will fix that!

Reply:
Done